packages/core/src/App/Containers/NotificationsDialog/notifications-dialog.jsx

```diff
@@ -30,6 +30,19 @@ const NotificationsList = ({ notifications, toggleDialog }) => {
         return `IcAlert${toTitleCase(type)}`;
     };
 
+    const getButtonSettings = item => {
+        if (!isEmptyObject(item.action)) {
+            return 'action';
+        } else if (!isEmptyObject(item.secondary_btn)) {
+            return 'secondary_btn';
+        } else if (!isEmptyObject(item.cta_btn)) {
+            return 'cta_btn';
+        } else if (!isEmptyObject(item.primary_btn)) {
+            return 'primary_btn';
+        }
```

Comment:
since we have return in each if, elses can be omitted 🙂
```suggestion
        if (!isEmptyObject(item.action)) return 'action';
        if (!isEmptyObject(item.secondary_btn)) return 'secondary_btn';
        if (!isEmptyObject(item.cta_btn)) return 'cta_btn';
        if (!isEmptyObject(item.primary_btn)) return 'primary_btn';
```
as an alternative, it can be refactored to:
```suggestion
        return ['action', 'secondary_btn', 'cta_btn', 'primary_btn'].find(el => !isEmptyObject(item[el]));
```